behat/tests/features-v2/reporting/sections/actions/actions.feature

```diff
@@ -0,0 +1,44 @@
+@v2
+Feature: Actions
+
+  Scenario: I complete the actions section with negative responses
+    Given a Lay Deputy logs in and has a new report
+    And I view and start the actions report section
+    Then I should be on the financial decision actions page
+    When I choose no and save on financial decision actions section
+    Then I should be on the concerns actions page
+    When I choose no and save on concerns actions section
+    Then I should be on the actions report last step page
+    And I should see the expected action report section responses
+    When I follow link back to report overview page
+    Then I should be on the Lay reports overview page
+    And I should see "actions" as "finished"
+
+  Scenario: I complete the actions section with positive responses
+    Given a Lay Deputy logs in and has a new report
+    And I view and start the actions report section
+    Then I should be on the financial decision actions page
+    When I choose yes and save on financial decision actions section
+    Then I should be on the concerns actions page
+    When I choose yes and save on concerns actions section
+    Then I should be on the actions report last step page
+    And I should see the expected action report section responses
+    And I should see the expected action comments
+
+  Scenario: I half fill in responses and come back and edit them
```

Comment:
I guess here the test is on editing existing responses so maybe something like `A user partially completes the section and then edits their responses` focuses the reader on what we're testing?